projects/go-lib/src/lib/components/go-select/go-select.component.html

```diff
@@ -39,6 +39,12 @@
         [ngClass]="{ 'go-select__select-all-button--dark' : theme === 'dark' }">
         Select All
       </button>
+      <button
+      (click)="onRemoveAll()"
+      class="go-select__select-all-button"
+      [ngClass]="{ 'go-select__select-all-button--dark' : theme === 'dark' }">
+      Remove  All
+    </button>
```

Comment:
Should we have a conditional here to only render the button if the value of the control is set? Not sure it makes sense from a UX perspective to show a "Remove All" option if they haven't selected anything.

Reply:
@grahamhency I have added showRemoveAll condition. Please check and let me know if it is okay or not

Reply:
@priyankafnu my question still stands. The change that you've put up to add the `showRemoveAll` binding here will always be true unless someone passes in the value to `go-select` of `true`. I don't think we need the `showRemoveAll` binding here. We can probably just do:
```
<ng-container *ngIf="multiple && items?.length">

Reply:
Also, we really should only show this button if the control has a value. So the final conditional should look like:
```
<ng-container *ngIf="multiple && items?.length && control.value">

Reply:
@priyankafnu, I still think that we should only render this button when the control has a value. Would you mind adding it to the conditional?

```
<ng-container *ngIf="multiple && items?.length && control.value">